src/core/algorithms/md/hymd/preprocessing/similarity_measure/date_dif_similarity_measure.h

```diff
@@ -0,0 +1,22 @@
+#include "algorithms/md/hymd/preprocessing/similarity_measure/distance_similarity_measure.h"
+#include "model/types/date_type.h"
+
+size_t DateDifference(const std::byte* left, const std::byte* right) {
```

Comment:
The metric functions you write should accept the proper type, not the byte pointer, as we may want to move them later, to use in other algorithms.
```suggestion
size_t DateDifference(model::Date const& left, model::Date const& right) {
```